src/composables/useNetwork.ts

```diff
@@ -1,17 +1,5 @@
 import { ref } from 'vue';
-
-/**
- * TYPES
- */
-export enum Network {
-  MAINNET = 1,
-  ROPSTEN = 3,
-  RINKEBY = 4,
-  GÖRLI = 5,
-  KOVAN = 42,
-  POLYGON = 137,
-  ARBITRUM = 42161
-}
+import { Network } from '@balancer-labs/sdk';
```

Comment:
Seems a bit weird to pull the network type from the SDK. I know the types will most likely be the same, but I think the network types available in the app should probably be defined in the app. I don't feel strongly about this, it's just a thought, feel free to keep it if you think it's the best path.